src/utils/folders.ts

```diff
@@ -21,29 +12,5 @@ export const doesFolderExist = async (folderId: string): Promise<boolean> => {
 };
 
 export const getAllNotesInFolder = async (folderId: string): Promise<Note[]> => {
-    let pageNum = 1;
```

Comment:
Is this function (and module) needed anymore?

Reply:
No, they're not thanks for pointing that out. If this is the only review comment, you can merge this pr, I'll remove these functions in the next pr.

Reply:
Good to see that we're on the same page. I'm looking forward to the next PR.